python/python/glide/async_commands/core.py

```diff
@@ -184,6 +184,11 @@ def get_cmd_args(self) -> List[str]:
         return [self.cmd_arg] if self.value is None else [self.cmd_arg, self.value]
 
 
+class InsertPosition(Enum):
```

Comment:
Is it auto-exported to a user?

Reply:
Users will have to import it, like they would for the other enums in this file as well (eg ExpireOptions).